### ISSUE TITLE
CLI should handle stdin and stdout

### ISSUE THREAD

**Issue by 1blankz7** [angstr0m/PaperBeard#11]
Like other Unix tools the default should work on stdin and stdout. For a more sophisticated operations we could add options to the application.

**Comment by 1blankz7** [comment]
Only if we specify a output and input the application should use this.

**Comment by angstr0m** [comment]
Agreed, but please think of the poor users that also use Windows ;). As long as compatibility with Windows isn't broken I'm happy.
